### PR TITLE
Release notes for Python Agent v9.1.1

### PR DESCRIPTION
**Pull request by lrafeei**
This PR adds release notes for Python Agent v9.1.1

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## New features
+
+* **Add caching to `get_package_version` to increase performance**
+The agent removes redundant calls to find the package version which will address performance issues.
+
+## Bug fixes
+
+* **Fix generator and async generator instrumentation issue in Redis**
+Fixed instrumentation of generators in `Redis.asyncio`.
```
Comment (ally-sassman) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx:25]: @lrafeei Can we combine these into just one sentence?

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Notes
+
+This release of the Python agent fixes generator instrumentation issue with Redis and adds caching to package versions to address some performance issues.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* **Add caching to `get_package_version` to increase performance**
+The agent removes redundant calls to find the package version which will address performance issues.
```
Comment (ally-sassman) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx:20]: @lrafeei Same with this one

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx (hunk 1 above)
Comment (lrafeei) [reply src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx:25]: Sure--I don't have strong feelings on the matter

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx (hunk 2 above)
Comment (lrafeei) [reply src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx:20]: Sounds good

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**